PasswordGenerator.py

```diff
@@ -0,0 +1,41 @@
+# The main purpose of this is to get secure password to defend against Bruteforce attacks.
```

Comment:
This repo requires that files are placed in an existing directory.  See CONTRIBUTING.md.